tpu_inference/layers/vllm/fused_moe.py

```diff
@@ -206,7 +205,7 @@ def expert_sharded_gmm(
     is_last_expert: bool,
     mesh: Mesh,
 ) -> jax.Array:
-    ep_size = mesh.shape["model"]
+    ep_size = mesh.shape["model"] * mesh.shape.get("attn_dp", 1)
```

Comment:
nit: i see this kind of patter frequently but it looks quite messy. do you know if python provides a there's a cleaner way to handle it? or maybe we can provide a helper function where
- Take list/tuple as an one argument
- Either string or list of string as another argument.
- If the argument is string perform regular key/value fetch
- If the argument is list of string, iterate of the strings and perform key/value fetch.